src/app/widgets/Visualizer/index.jsx

```diff
@@ -964,36 +963,20 @@ class VisualizerWidget extends PureComponent {
-                yAxisColor: this.config.get('Custom Light Y Axis') ? this.config.get('Custom Light Y Axis') : LIGHT_THEME_VALUES.yAxisColor,
-                zAxisColor: this.config.get('Custom Light Z Axis') ? this.config.get('Custom Light Z Axis') : LIGHT_THEME_VALUES.zAxisColor,
-                limitColor: this.config.get('Custom Light Limit') ? this.config.get('Custom Light Limit') : LIGHT_THEME_VALUES.limitColor,
-                cuttingCoordinateLines: this.config.get('Custom Light Cutting Coordinate Lines') ? this.config.get('Custom Light Cutting Coordinate Lines') : LIGHT_THEME_VALUES.cuttingCoordinateLines,
-                joggingCoordinateLines: this.config.get('Custom Light Jogging Coordinate Lines') ? this.config.get('Custom Light Jogging Coordinate Lines') : LIGHT_THEME_VALUES.joggingCoordinateLines,
-                G0Color: this.config.get('Custom Light G0') ? this.config.get('Custom Light G0') : LIGHT_THEME_VALUES.G0Color,
-                G1Color: this.config.get('Custom Light G1') ? this.config.get('Custom Light G1') : LIGHT_THEME_VALUES.G1Color,
-                G2Color: this.config.get('Custom Light G2') ? this.config.get('Custom Light G2') : LIGHT_THEME_VALUES.G2Color,
-                G3Color: this.config.get('Custom Light G3') ? this.config.get('Custom Light G3') : LIGHT_THEME_VALUES.G3Color,
+                backgroundColor: this.config.get('Custom Background') ? this.config.get('Custom Background') : DARK_THEME_VALUES.backgroundColor,
+                gridColor: this.config.get('Custom Grid') ? this.config.get('Custom Grid') : DARK_THEME_VALUES.gridColor,
+                xAxisColor: this.config.get('Custom X Axis') ? this.config.get('Custom X Axis') : DARK_THEME_VALUES.xAxisColor,
+                yAxisColor: this.config.get('Custom Y Axis') ? this.config.get('Custom Y Axis') : DARK_THEME_VALUES.yAxisColor,
+                zAxisColor: this.config.get('Custom Z Axis') ? this.config.get('Custom Z Axis') : DARK_THEME_VALUES.zAxisColor,
+                limitColor: this.config.get('Custom Limit') ? this.config.get('Custom Limit') : DARK_THEME_VALUES.limitColor,
+                cuttingCoordinateLines: this.config.get('Custom Cutting Coordinate Lines') ? this.config.get('Custom Cutting Coordinate Lines') : DARK_THEME_VALUES.cuttingCoordinateLines,
+                joggingCoordinateLines: this.config.get('Custom Jogging Coordinate Lines') ? this.config.get('Custom Jogging Coordinate Lines') : DARK_THEME_VALUES.joggingCoordinateLines,
+                G0Color: this.config.get('Custom G0') ? this.config.get('Custom G0') : DARK_THEME_VALUES.G0Color,
+                G1Color: this.config.get('Custom G1-3') ? this.config.get('Custom G1-3') : DARK_THEME_VALUES.G1Color,
+                G2Color: this.config.get('Custom G1-3') ? this.config.get('Custom G1-3') : DARK_THEME_VALUES.G2Color,
+                G3Color: this.config.get('Custom G1-3') ? this.config.get('Custom G1-3') : DARK_THEME_VALUES.G3Color,
```

Comment:
Like i mentioned in another comment, you can pass a default value as the second argument in .get instead of using the ternary operator. Just makes the code more maintainable and more clean